index.html

```diff
@@ -35,6 +35,14 @@ <h2 class="page-heading">Knossos.NET Downloads</h2>
       </div>
     </header>
 
+    <div class="container">
```

Comment:
Containers are bootstrap classes which allow for responsive subelements, "row" and then "col", which make a grid type system with 12 columns total.  As it is now, this would probably display identically with or without the container tag.

Reply:
Without the container tag, the placement isn't correct.  I tried it.  The container probably includes some sort of margin or spacing attributes.